recipe/migrations/pypy38.yaml

```
@@ -0,0 +1,52 @@
+migrator_ts: 1647123563
+__migrator:
+    migration_number: 1
+    primary_key: python
+    ordering:
+        python:
+            - 3.6.* *_cpython
+            - 3.7.* *_cpython
+            - 3.8.* *_cpython
+            - 3.9.* *_cpython
+            - 3.10.* *_cpython
+            - 3.6.* *_73_pypy
+            - 3.7.* *_73_pypy
+            - 3.8.* *_73_pypy
+            - 3.9.* *_73_pypy
+    paused: False
+    longterm: True
+    check_solvable: True
+    exclude_pinned_pkgs: False
+    pr_limit: 1
+    bump_number: 1
+    wait_for:
+      - python310
```

OTOH, I think this needs to be:
```suggestion
    wait_for_migrators:
      - python310
```
cf. [search](https://github.com/conda-forge/conda-forge-pinning-feedstock/search?q=wait_for_migrators) on this repo (and no results for `wait_for`).